HW_1/Task_4.py

```diff
@@ -0,0 +1,14 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов.
+# Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали
+# одинаковое количество журавликов, а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+# Петя сделал а журавликов, Сережа сделал b журавликов, Катя сделала (a + b)**2
+# вместе они сделали a + b + (a + b)**2 = sum или 2a + 4a**2 = sum, преобразовав уравнение получаем a=sum/6
+
+sum = int(input('Введите сумму сделанных журавликов, кратную 6: '))
+while (sum % 6 != 0 or sum < 0):
+    sum = int(input('Вы ввели не корректные данные. '
+                    'Введите сумму сделанных журавликов, кратную 6: '))
+else:
+    print(f"Петя сделал {round(sum / 6)} журавликов, "
+          f"Сережа сделал {round(sum / 6)}, Катя сделала {round(sum / 6 * 4)}")
```

Comment:
Зачем использовать функцию округления, если округлять будет нечего? Просто используйте оператор целочисленного деления. Но перед этим лучше создать переменную с повторяемым выражением.
```suggestion
    res = sum // 6
    print(f"Петя сделал {res} журавликов, "
          f"Сережа сделал {res}, Катя сделала {res * 4)}")
```